boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -44,6 +50,83 @@
 		led0 = &green_led_4;
 		sw0 = &user_button;
 	};
+
+	fmc: memory-controller@A0000000 {
+		compatible = "st,stm32-fmc";
+		reg = <0xA0000000 0x1000>;
+		clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00000001>;
+		label = "STM32_FMC";
+		status = "disabled";
```

Comment:
All of the above needs to go at soc level

Reply:
Ok. So all the low level generic stuff on soc level and then timing info and pinctrl on board level since it would depend on ram chip. Correct? Or is it better to leave defaults in place on soc level?